classes/capquiz.php

```diff
@@ -82,6 +82,10 @@ public function is_grading_completed() : bool {
         return $this->record->timedue < time() && $this->record->timedue > 0;
     }
 
+    public function duedate() : string {
+        return date('d-m-Y', $this->record->timedue);
+    }
+
```

Comment:
Remove this function in favor for the Moodle guidelines. This way, the date won't be localized.